open-hackathon-adminUI/src/app/admin/login_background.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding:utf8 -*-
+# encoding = utf-8
+import sys
+import urllib2
+
+sys.path.append("..")
+from app.functions import get_remote, get_config, convert
+from app.log import log
+from user_mgr import user_manager
+import json
+from app.constants import OAUTH_PROVIDER
+
+
+class LoginUser:
```

Comment:
I'm confused by the two duplicated `LoginUser`. They are of the same code. What's the purpose?